frontend/src/components/molecules/DetailView.tsx

```diff
@@ -7,9 +8,59 @@ import {
   getDevelopmentArea,
   getTradeOff,
 } from '../../api/ontologies';
+import { mapCorrelationToName } from '../../common/node';
 import { RootState } from '../../state/store';
 import { Annotation, Node } from '../../types/ontologyTypes';
 import Connections from '../atoms/Connections';
+import SlideInDrawer from '../atoms/SlideInDrawer';
+
+interface ContextDividerProps {
+  visible: boolean;
+}
+
+const ContextDivider: React.FC<ContextDividerProps> = ({ visible }: ContextDividerProps) => {
+  if (!visible) return <></>;
+  return (
+    <Center mx="20">
+      <Divider orientation="vertical" />
+    </Center>
+  );
+};
+
+interface AllConnectionsProps {
+  contributions: Array<Node>;
+  tradeOffs: Array<Node>;
+  developmentAreas: Array<Node>;
+  onClick: (connection: Node) => void;
+}
+
+const AllConnections: React.FC<AllConnectionsProps> = ({
```

Comment:
Is there any specific reason why we are defining 3 components in this file? I get that we most likely aren't going to re-use them in other places as they are kind of niche, but having them as separate file shouldn't hurt. Separating them out would de-clutter the file and make it easier to do changes at a later stage.